samples/plot_peptide_cpH.py

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright (C) 2024 pyMBE-dev team
+#
+# This file is part of pyMBE.
+#
+# pyMBE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyMBE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Plots the charge of a peptide with a given sequence using either a custom or reference set of pKa values
+
+import numpy as np
+import matplotlib.pyplot as plt
+import argparse
+import pandas as pd
+# Create an instance of pyMBE library
+import pyMBE
+pmb = pyMBE.pymbe_library(seed=42)
+
+parser = argparse.ArgumentParser(description='Plots the titration data from peptide_mixture_grxmc_ideal.py and the corresponding analytical solution.')
+parser.add_argument('--sequence',
+                    type=str,
+                    default= 'EEEEDDDD', 
+                    help='sequence of the peptide')
+parser.add_argument('--path_to_data',
+                    type=str,
+                    required= False,
+                    default="samples/time_series/peptide_cpH/analyzed_data.csv",
```

Comment:
The default folder name should be ´peptide' instead of ´peptide_cpH' to be consistent with the production script

Reply:
Actually, I decided to solve this issue the other way around by consistently naming all folders "peptide_cpH" to clearly link it to the script that produced the data.